.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -38,3 +38,7 @@ Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.
 [user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 output here
 ```
+
+### Prioritisation
+
+Add :+1: for [issues you find important](https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).
```

Comment:
We didn't decide on a subheading, and also I have no idea how it eluded us that it's "add x to y" and not "add x for y", unless *y* is the purpose of doing the addition.

```suggestion
### Priorities

Add :+1: to [issues you find important](https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).
```

Reply:
![priorities](https://user-images.githubusercontent.com/6599296/205128490-b758f963-386a-4b28-82a1-cf6ce88cc904.jpg)

Reply:
Not a big fan of this but done :)